SPECS/abseil-cpp/abseil-cpp.spec

```diff
@@ -77,7 +72,7 @@ ctest --output-on-failure
 %files
 %license LICENSE
 %doc FAQ.md README.md UPGRADES.md
-%{_libdir}/libabsl_*.so.2111.*
+%{_libdir}/libabsl_*.so.2206.*
```

Comment:
Is the explicit number required here?

Reply:
Not really but it's preferred - it helps to immediately see that we might need to bump up dependents packages. In this case it was `grpc`.